### PR TITLE
Update a whole lot of things to "modernize" the plugin structure

### LINKED ISSUE #2: Needs to account for focusable elements for accessibility
If a section contains a focusable element, I'm pretty sure we need to manually change the focus to that element when the scroll animation occurs for accessibility reasons.

### PR DESCRIPTION
**Pull request by jeremyfelt**
Among other things, fixes #2

### CODE REVIEW

File: blocks/scrolling-section/block.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "happyprime/scrolling-section",
+	"title": "Scrolling Section",
+	"category": "design",
+	"icon": "media-default",
+	"description": "An individual full-height scrolling section",
+	"keywords": [ "toggle", "faq", "show hide", "accordion" ],
```
Comment (philcable) [inline blocks/scrolling-section/block.json]: I can't think of any keywords that actually fit with this block off the top of my head, so we can maybe just remove this 🙂.

File: blocks/scrolling-sections/block.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "happyprime/scrolling-sections",
+	"title": "Scrolling Sections",
+	"category": "design",
+	"icon": "iamge-flip-vertical",
```
Comment (philcable) [inline blocks/scrolling-sections/block.json]:
```suggestion
	"icon": "image-flip-vertical",
```

File: blocks/scrolling-sections/block.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	"description": "Create full-height scrolling sections on the page",
+	"keywords": [],
+	"version": "1.0.0",
+	"textdomain": "show-hide-section",
```
Comment (philcable) [inline blocks/scrolling-sections/block.json]:
```suggestion
	"textdomain": "scrolling-sections",
```

File: package.json [hunk 4]
```diff
@@ -4,18 +4,31 @@
...
     "check-engines": "wp-scripts check-engines",
     "check-licenses": "wp-scripts check-licenses --production",
     "lint:css": "wp-scripts lint-style '**/*.css'",
-    "lint:js": "wp-scripts lint-js 'src/*.js'",
-    "lint:pkg-json": "wp-scripts lint-pkg-json .",
-    "start": "wp-scripts start",
-    "test:e2e": "wp-scripts test-e2e",
-    "test:unit": "wp-scripts test-unit-js"
+    "lint:js": "wp-scripts lint-js 'blocks/**/*.js'",
+	"lint:fix": "wp-scripts lint-js 'blocks/**/*.js' --fix",
```
Comment (philcable) [inline package.json]:
```suggestion
    "lint:fix": "wp-scripts lint-js 'blocks/**/*.js' --fix",
```

**Review by philcable — COMMENTED** [review]
Looking good! Just a little bit of feedback on some small things.

File: blocks/scrolling-section/block.json (hunk 1 above)
Comment (jeremyfelt) [reply blocks/scrolling-section/block.json]: lol, nice copy and paste @jeremyfelt 🤦🏻‍♂️

Comment (jeremyfelt) [inline blocks/scrolling-section/block.json]:
```suggestion
```

**Pull request merged**